.pre-commit-config.yaml

```diff
@@ -29,8 +29,14 @@ repos:
   #   -   id: black
   - repo: local
     hooks:
-      - id: nox-flake8
-        name: nox-flake8
+      - id: nox-black
+        name: nox-black
+        entry: cd ./backend/ && pipenv run nox -s black
+        language: system
+        types: [python]
+        pass_filenames: false
```

Comment:
Is there a reason you want Black to actually reformat your files during a pre-commit hook?  The `lint` session already checks to see whether your files are Black formatted, but it just doesn't format them.

The way you have it here it will reformat your files, but those changes would be unstaged.  And depending on timing, the `nox-lint` pre-commit hook could run before or after reformat and either find no issues (because the Black formatter already ran due to `nox-black` pre-commit hook) or still find the issues.

Reply:
removed nox-black